alembic/versions/9b188aee9577_add_new_tech_tables.py

```diff
@@ -0,0 +1,47 @@
+"""add_new_tech_tables
+
+Revision ID: 9b188aee9577
+Revises: 0fda0f2746d6
+Create Date: 2024-05-03 15:28:55.342488
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import declarative_base
+
+# revision identifiers, used by Alembic.
+revision = "9b188aee9577"
+down_revision = "0fda0f2746d6"
+branch_labels = None
+depends_on = None
+Base = declarative_base()
+
+
+def upgrade():
+    op.create_table(
+        "run_device",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("type", sa.String(128), nullable=False),
```

Comment:
I don't think we need 128 characters, and 32 is the one that's most used in other columns
```suggestion
        sa.Column("type", sa.String(32), nullable=False),
```